src/patternfly/utilities/BackgroundColor/BackgroundColor.scss

```diff
@@ -19,9 +19,25 @@ $pf-u-background-color-options: (
   ),
   background-color-dark-400: (
     background-color var(--pf-global--BackgroundColor--dark-400)
+  ),
+  background-color-danger: (
+    background-color var(--pf-global--palette--red-50)
```

Comment:
Since we're using these as background colors for states, I wonder if these should be global vars. All of the colors for text and background colors currently map to global vars. @mceledonia wdyt? It wouldn't necessarily have to be a part of this PR though.